docs/guides/troubleshooting.md

```diff
@@ -23,12 +23,17 @@ do not require such adjustment.
 
 XCUITest driver offers a few methods to handle them.
 
+- Start a session without `appium:app` nor `appium:bundleId`. Then XCUITest driver attempts to get the current active application. This requires you to start an application after a new session request with [`mobile: installApp`](../reference/execute-methods.md#mobile-installapp) to install an app if needed and [`mobile: launchApp`](../reference/execute-methods.md#mobile-launchapp)/[`mobile: activateApp`](../reference/execute-methods.md#mobile-activateapp), but it could automatically change the active application with `com.apple.springboard` or activated application on the top.
```

Comment:
this basically simulates the legacy approach, with on-screen pixel autodetection (which could be slow).

Reply:
Yea, i have addressed the possible delay as a note.